scenes/enumerated/problem_10320/problem_10320.py

```diff
@@ -0,0 +1,207 @@
+        self.play(FadeIn(city))
+        self.wait()
+        
+        train = Train(number_of_rolling_cars=1).scale(.2).next_to(road.left_edge, UP).scale(.8)
+        self.play(FadeIn(train))
+        self.wait()
+        
+        self.play(Arsen.animate.scale(.2).move_to(train.get_center()))
+        self.play(FadeOut(Arsen))
+
+        first_part_of_road = self.get_part(value=320).scale(scale).align_to(road.line, LEFT+UP)
+        stopwatch = Stopwatch()
+        stopwatch.scale(.5).to_edge(UP+RIGHT).shift(LEFT)
+        timer = always_redraw(lambda: MathTex("{}".format(floor(stopwatch.time.get_value()/240*4))).next_to(stopwatch, DOWN))
+        self.add(timer)
+        self.play(
+            train.animate.next_to(first_part_of_road.right_edge, UP),
+            Run(stopwatch, speed=60, run_time=4),
+            run_time=4,
+            rate_func=linear
+        )
+        self.play(FadeIn(first_part_of_road))
+        first_part_of_road_time = MathTex(first_part_of_road_time_string).next_to(first_part_of_road, UP, buff=.5)
+        self.play(Write(first_part_of_road_time))
+        self.wait()
+        second_part_of_road = self.get_part(value=285).scale(scale).next_to(first_part_of_road, buff=0)
+        self.play(Reset(stopwatch))
+        self.play(
+            train.animate.next_to(second_part_of_road.right_edge, UP),
+            Run(stopwatch, speed=60, run_time=3),
+            run_time=3,
+            rate_func=linear
+        )
+        self.add(second_part_of_road)
+        second_part_of_road_time = MathTex(second_part_of_road_time_string).next_to(second_part_of_road, UP, buff=.5)
+        self.play(Write(second_part_of_road_time))
+        self.wait()
+
+        self.play(
+            VGroup(
+                road,
+                first_part_of_road,
+                first_part_of_road_time,
+                second_part_of_road,
+                second_part_of_road_time,
+                road_length,
+                train,
+                city
+            ).animate.to_edge(DOWN, buff=.8),
+            FadeOut(stopwatch, timer)
+        )
+        self.play(FadeOut(train, city))
+        self.wait()
+        
+        one_part = self.get_part(value=80, color=GREEN).scale(scale).to_edge(LEFT).to_edge(UP, buff=1.7)
+        self.play(GrowFromEdge(one_part, LEFT))
+        self.wait()
+        
+        one_part_copy = one_part.copy()
+        self.play(FadeIn(one_part_copy))
+        self.play(one_part_copy.animate.shift(DOWN*1.5))
+        self.wait()
+        segment_15 = self.get_part(15, color=YELLOW).scale(scale).next_to(one_part_copy, buff=0)
+        segment_15.set_label(MathTex(segment_15_string))
+        segment_15_label = segment_15.update_label_pos(buff=.15)
+        self.play(
+            GrowFromEdge(segment_15, LEFT),
+            Write(segment_15_label)
+        )
+        self.wait()
+
+        x4_one_part = [*[one_part.copy() for i in range(4)]]
```

Comment:
```suggestion
        x4_one_part = [one_part.copy() for i in range(4)]
```

Is there a difference?